src/aviary/utils.py

```diff
@@ -211,6 +212,37 @@ async def extract_answer(
     return None
 
 
+class RandomAnnotation:
```

Comment:
Out of curiosity, why go this route instead of `field_serializer`/`validator`?

Reply:
Yeah good question, it's for reusability. I can easily import/reuse `RandomAnnotation` across many different `BaseModel`, whereas `field_serializer`/`validator` are class-specific and not reusable